apps/studio/components/interfaces/Auth/BasicAuthSettingsForm/BasicAuthSettingsForm.tsx

```diff
@@ -84,19 +83,19 @@ const BasicAuthSettingsForm = () => {
     }
   }, [authConfig])
 
-  const onSubmit = (values: any) => {
-    const payload = { ...values }
-    payload.DISABLE_SIGNUP = !values.DISABLE_SIGNUP
-    // The backend uses empty string to represent no required characters in the password
-    if (payload.PASSWORD_REQUIRED_CHARACTERS === NO_REQUIRED_CHARACTERS) {
-      payload.PASSWORD_REQUIRED_CHARACTERS = ''
-    }
```

Comment:
this key was never in the form's values so the condition would always be false

if it's necessary for the payload I can add back in the empty string